### PR TITLE
heroku app homework submittal

### PR DESCRIPTION
**Pull request by joeelsey**
Not the prettiest app, but it works.  Was a totally brutal process to get from the server.js portion to the html portion.  I spent more time on the front end than on the back end of this project.  Please let me know if you have any questions or concerns.  Thanks.

### CODE REVIEW

File: public/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  var ZipCode = 1;
+  $('#SetZipCode').click(function() {
+    var ZipCode = document.getElementById('GetZipCode').value;
+    $('#GetZipCode').val(" ");
+    console.log(ZipCode);
+
+    $.ajax({
+      type: 'GET',
+      dataType: 'text',
+      url: 'https://coldenoughforbeer.herokuapp.com/' + 'zip/' + ZipCode,
```
Comment (toastynerd) [inline public/index.js]: There's not really a need for the + 'zip', you should just add it to the previous value.

File: public/index.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      success: function(data) {
+        $('#answer').html(data);
+      }
+    });
+
+  });
+});
+//url: 'https://localhost:3000/zip/'
```
Comment (toastynerd) [inline public/index.js]: Don't leave commented out code in your app, just get rid of it.

File: server.js [hunk 3]
```diff
@@ -1,18 +1,66 @@
...
+app.get('/zip/:zip', function(req, res) {
+  //var zip = document.getElementById('zip').value;
+  var key = 'b40a0fca31859481';
+  var weatherUrl = 'http://api.wunderground.com/api/' + key +
+    '/conditions/q/' + req.params.zip + '.json';
 
   request
     .get(weatherUrl)
     .end(function(err, weatherdata) {
       if (err) res.status(500).send(err);
```
Comment (toastynerd) [inline server.js:30]: you should probably return here as well as sending the 500.

File: server.js [hunk 4]
```diff
@@ -1,18 +1,66 @@
...
 
-app.get('/', function(req, res) {
-  var weatherUrl = "http://www.wunderground.com/?apiref=fca0fd6081216f5b";
+/*app.get('/', function(req, res) {
+  res.json({
+    msg: 'add zip code to the url to get temp.  Ex: /zip/yourzip'
+  });
+});*/
+
+app.get('/zip/:zip', function(req, res) {
```
Comment (toastynerd) [inline server.js:22]: Once again, get rid of commented out code.

File: server.js [hunk 5]
```diff
@@ -1,18 +1,66 @@
+'use strict';
+
 var express = require('express');
 var request = require('superagent');
 var app = express();
+var createDomain = require('domain').create;
+
+//https://speakerdeck.com/felixge/domains-in-node-08
+app.use(function(req, res, next) {
+  var domain = createDomain();
```
Comment (toastynerd) [inline server.js:10]: While domains are cool, you shouldn't need one for this assignment. It's kind of code smell when someone is catching all errors when they don't really need to. Plus, domains don't 100% yet.

**Comment by toastynerd** [comment]
Got a link to your travis build?

**Comment by joeelsey** [comment]
I didn't do any testing for the app.  I ran out of time before weekend and
spent Sat & Sun working on JWT.  I can add testing this week if needed.

**Comment by toastynerd** [comment]
You have a test in the PR...

**Comment by toastynerd** [comment]
There really doesn't have to be a ton of testing for this app, just turn on travis and send me a link. =)

**Comment by joeelsey** [comment]
I do.  I completely forgot about that.  It only tests to see if the api
doesn't return undefined.  I can add a travis badge today.

**Comment by joeelsey** [comment]
Will do.  Thank you.

**Comment by joeelsey** [comment]
https://github.com/joeelsey/isitcoldenoughoutside/tree/branch1

Here's my repo with the commented code cleaned up and the travis badge.
Thanks.

**Pull request closed without merging**